### ISSUE TITLE
font-files helper: Add woff2 format

### ISSUE THREAD

**Issue by mechanicalduck** [Compass/compass#1756]
woff2 font format has been recently introduced and is currently used by Google Chrome 
and preferably served to Google Chrome (e.g. by Google Fonts API).

The font-files helper doesn't support it woff2 and fails when specifying it in the given list:

```
@include font-face(
  "Test Regular",
  font-files(
    "Test.woff2", woff2,
    "Test.woff",  woff,
    "Test.ttf",   ttf,
    "Test.svg#Test", svg
  ),
  "Test.eot",
  normal,
  normal,
  (U+1700-1714, U+1735-1736),
  "Test"
);
```

```
error [...] ([...]: Could not determine font type for Test.woff2)
```

With best regards

**Comment by mprins** [comment]
duplicate of / fixed in #1742

**Comment by chriseppstein** [comment]
fixed
